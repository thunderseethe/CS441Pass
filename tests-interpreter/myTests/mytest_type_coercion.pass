begin
	print 1 + 2 + 3 + 4;
	print 355.0/113.0 - (22.0/7.0 + 1.0) + 0.3 + 0.7;
	print 355/113 + 1.0 + (6.0 -1.33)*4.1;
end