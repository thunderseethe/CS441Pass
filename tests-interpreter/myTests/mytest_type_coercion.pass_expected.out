10
-0.001264
23.147000
